### PR TITLE
Update Discord.js to v13.1.0

### PR DESCRIPTION
**Pull request by sujalgoel**

### CODE REVIEW

**Review by MrAugu — CHANGES_REQUESTED** [review]
For the coding style; the double quotes and double spacing indentation should also.

**Review by MrAugu — APPROVED** [review]
The changes look good to me.

**Comment by MrAugu** [comment]
Before merging this pull request, a review is still pending from @Myst82015.

**Comment by sujalgoel** [comment]
> Before merging this pull request, a review is still pending from @Myst82015.

ok

File: index.js [hunk 1]
```diff
@@ -1,82 +1,93 @@
...
-    const roundtripMessage = await reply("Pong!");
-    return roundtripMessage.edit(`*${roundtripMessage.createdTimestamp - message.createdTimestamp}ms*`);
-  }
+	// If command is ping we send a sample and then edit it with the latency.
+	if (command === 'ping') {
+		const roundtripMessage = await message.channel.send({ content: 'Pong!' });
+		return roundtripMessage.edit(
+			`*${roundtripMessage.createdTimestamp - message.createdTimestamp}ms*`,
+		);
+	}
```
[the remark this answers is not in the document]
Comment (MrAugu) [reply index.js]: We kinda should.

[the remark this answers is not in the document]
Comment (MrAugu) [reply index.js]: But at another time.

**Review by Myst82015 — APPROVED** [review]

**Pull request merged**